src/routes/index.js

```diff
@@ -0,0 +1,16 @@
+import Feed from "./Feed/Feed";
+import FindPeers from "./FindPeers/FindPeers";
+import Landing from "./Landing/Landing";
+import Login from "./Login/Login";
+import Signup from "./Signup/Signup";
+import UserProfile from "./UserProfile/UserProfile";
+
+
+export{
+    Feed,
+    FindPeers,
+    Landing,
+    Login,
+    Signup,
+    UserProfile
+}
```

Comment:
Instead of an explicit export statement here, you can replace import with export in lines 1-6